### PR TITLE
Added a timezone specification

### PR DESCRIPTION
**Pull request by hitohata**
## Overview of Implementation

Added a timezone specification

### CODE REVIEW

**Review by uskayyyyy — COMMENTED** [review]
LGTM :)
Let's wait for someone else's review, as well!

**Comment by hitohata** [comment]
@kotaaaa tells me this one needs to be written the communication between the frontend and backend is done by ISO  8601.

File: project-document/docs/specifications/others.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Others
+---
+
+## Timezone
+
```
Comment (kotaaaa) [inline project-document/docs/specifications/others.md:6]: Something like this make sense more, what do you think?
`Follow the ISO format, including the time zone.`

Comment (hitohata) [reply project-document/docs/specifications/others.md:6]: I've added mentioning it.

**Comment by hitohata** [comment]
I've updated the documents, please check it.

**Review by kotaaaa — APPROVED** [review]
Thank you for updating!

**Pull request merged**